tests/VolleyManagement.UnitTests/Services/GameService/GameServiceTests.cs

```diff
@@ -1331,6 +1331,37 @@ public void Create_FifthSetValidScore_GameCreated()
             VerifyCreateGame(newGame, Times.Once());
         }
 
+        [TestMethod]
+        public void Create_SameTeamInTheRound_ExceptionThrown()
```

Comment:
Are you sure there is no other test with same/similar name testing this scenario?

Reply:
Test setup does not use any notion of Round

Reply:
Yes, I am sure. As proof you can add this TestMethod to the GameServiceTests.cs on master branch and make sure that only my test will crash.
Notation of Round is used in GameService.ts so you have just to belive me :)
UPD:
Round is setted up in TestGamesForDuplicateSchemeTwo method (as at the screenshot below
![image](https://user-images.githubusercontent.com/36235893/36499751-ee2b5d74-174a-11e8-9ab9-5c41a5352135.png)

Reply:
Also, Round for new game is setted up in TestFreeDayGame method.
![image](https://user-images.githubusercontent.com/36235893/36501014-959ad262-174e-11e8-82c0-dd1392830444.png)

Reply:
To make sure that my test methods are necessary you can switch to master branch and run all tests. You will see that no one test is crushing. But features written by Nataliia are not implemented here yet.